bigbluebutton-html5/imports/api/presentations/server/modifiers/addPresentation.js

```diff
@@ -6,10 +7,20 @@ import flat from 'flat';
 import addSlide from '/imports/api/slides/server/modifiers/addSlide';
 import setCurrentPresentation from './setCurrentPresentation';
 
+const getSlideText = async (url) => {
+  const file = await HTTP.get(url);
+
+  return file;
```

Comment:
What happens if the slide has no [obtainable] text? - i.e. image slide

Reply:
The file is generated even for slides without texts.
In this case the content is empty.